project_national/panel_upgrades.yml

```diff
@@ -0,0 +1,1366 @@
+      - &4_electric_resistance_heating_heating_type_and_fuel_wall_floor_furnace
+        option: HVAC Heating Type And Fuel|Electricity Electric Wall Furnace
+        apply_logic: &apply_logic_fuel_wall_floor_furnace
+          - or: 
+            - HVAC Heating Efficiency|Fuel Wall/Floor Furnace, 60% AFUE
+            - HVAC Heating Efficiency|Fuel Wall/Floor Furnace, 68% AFUE
+      - &5_electric_resistance_heating_heating_type_and_fuel_shared_heating
+        option: HVAC Heating Type And Fuel|Electricity Shared Heating
+        apply_logic:
+          - HVAC Heating Efficiency|Shared Heating 
+
+      - &6_electric_resistance_heating_heating_eff_boiler 
+        option: HVAC Heating Efficiency|Electric Boiler, 100% AFUE
+        apply_logic: *apply_logic_fuel_boiler      
+      - &7_electric_resistance_heating_heating_eff_fuel_furnace
+        option: HVAC Heating Efficiency|Electric Furnace, 100% AFUE
+        apply_logic: *apply_logic_fuel_furnace      
+      - &8_electric_resistance_heating_heating_eff_wall_floor_furnace
+        option: HVAC Heating Efficiency|Electric Wall Furnace, 100% AFUE
+        apply_logic: *apply_logic_fuel_wall_floor_furnace
+
+      - &9_electric_resistance_hvac_shared_eff_shared_heating_only
+        option: HVAC Shared Efficiencies|Boiler Baseboards Heating Only, Electricity
+        apply_logic:
+          - HVAC Heating Efficiency|Shared Heating
+          - not: HVAC Cooling Efficiency|Shared Cooling
+      - &10_electric_resistance_hvac_shared_eff_shared_heating_cooling
+        option: HVAC Shared Efficiencies|Fan Coil Heating and Cooling, Electricity
+        apply_logic:
+          - HVAC Heating Efficiency|Shared Heating
+          - HVAC Cooling Efficiency|Shared Cooling
+
+    #### 1.2 Federal Minimum Heat Pump With Electric Backup, 2 options  ####
+      - &1_fed_min_ashp_ducted
+        option: HVAC Heating Efficiency|ASHP, SEER 15.05, 8.82 HSPF, Max Load # same as base option
+        apply_logic:
+          - HVAC Has Ducts|Yes
+          - not: Heating Fuel|None
+          - not: HVAC Heating Type And Fuel|Electricity ASHP
+        costs: &costs_ashp_ducted_8-82_HSPF
+        # Source: Custom regression by Brennan Less. Ducted heat pump project costs were regressed on nameplate tons and HSPF,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. January 2019 $ inflated to April 2023 $ using 1.21 factor.
+        # Ducted ASHP regression n=317. Implicitly includes electrical upgrades and electric backup.
+          - value: 3680.19 # for HSPF 8.82
+            multiplier: Fixed (1)
+          - value: 167.64
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: &lifetime_HP 15
+
+      - &2_fed_min_mshp_ductless
+        option: HVAC Heating Efficiency|MSHP, SEER 14.5, 8.33 HSPF, Max Load # federal minimum is SEER 14.3, but 14.5 is lowest SEER in baseline ResStock
+        apply_logic:
```

Comment:
*logic_ductless_hvac_no_mshp

Reply:
ditto, if use the reference, got an error when validate the yml file.